api/swagger-spec/v1.json

```diff
@@ -19005,6 +19005,10 @@
        "$ref": "v1.HostAlias"
       },
       "description": "HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts file if specified. This is only valid for non-hostNetwork pods."
+     },
+     "priorityName": {
+      "type": "string",
+      "description": "If specified, indicates the pod's priority. `system` is a special keyword which indicates the highest priority. Any other name must be defined in Admission Controller config or pod will be rejected. If not specified, the pod priority will be zero."
```

Comment:
Juts nit: Those should be the generated files? As you are going to have a new patch based on the new proposal, so I think that you will update here also.